code/datums/weather/weather_types/snow_storm.dm

```diff
@@ -32,13 +36,26 @@
 /datum/weather/snow_storm/weather_act(mob/living/living)
 	living.adjust_bodytemperature(-rand(cooling_lower, cooling_upper))
 
+/// Tracks where we should play snowstorm sounds for the area sound listener
+GLOBAL_LIST_EMPTY(snowstorm_sounds)
```

Comment:
Probably want to put this with the other weather lists in:

https://github.com/tgstation/tgstation/blob/5ccbbf44448e4869df592cc5e05427e5a6abaca1/code/__DEFINES/weather.dm#L1-L7